.github/workflows/deploy-earn-protocol-landing-page-staging.yaml

```
@@ -0,0 +1,125 @@
+          version: 8.14.1
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: 'pnpm'
+
+      - name: Extract commit hash
+        id: vars
+        shell: bash
+        run: |
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)